numba/cuda/tests/cudapy/test_cooperative_groups.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import print_function
+
+import numpy as np
+
+from numba import config, cuda, int32, void
+from numba.cuda.testing import unittest, CUDATestCase, skip_on_cudasim
+
+
+def cc_X_or_above(major, minor):
+    if not config.ENABLE_CUDASIM:
+        cc = cuda.current_context().device.compute_capability
+        return cc >= (major, minor)
+    else:
+        return True
+
+
+def skip_unless_cc_60(fn):
+    return unittest.skipUnless(cc_X_or_above(6, 0), "requires cc >= 6.0")(fn)
```

Comment:
Should these go into a `numba.cuda.tests.utils` or similar, there's been a few of these of late?

Reply:
There's similar things in `numba.cuda.testing`, so I've moved them there.